source/_components/plugwise.markdown

```diff
@@ -0,0 +1,161 @@
+ha_iot_class: Local Polling
+ha_release: 0.98
+---
+
+This enables [Plugwise](https://plugwise.com) [Anna](https://www.plugwise.com/en_US/products/anna) thermostats to be integrated. This integration talks locally to your **Smile** interface, you will need its secret key and IP address.
+
+To find your IP address using the Plugwise App. When connected choose the 'Settings'-icon and go to 'HTML-interface'. Go to the (now little lower) 'Settings'-icon and choose 'Preferences'. Under 'System' then 'Networking' you will find your IP address.
+
+## Configuration
+
+You have to add the following to your configuration.yaml file.
+
+```yaml
+# Minimal configuration.yaml entry
+climate:
+  - platform: plugwise
+    password: your_short_id # required, the ID on the smile (some string of 6 characters)
+    host: local_ip_address  # required, the IP-address of your smile
+```
+
+{% configuration %}
+password:
+  description: Your Smile ID (located on the bottom of the Smile, not the Anna)
+  required: true
+  type: string
+host:
+  description: The IP address of your Smile. 
+  required: true
+  type: string
+name:
+  description: The name of your thermostat, i.e. "Anna"
+  required: false
+  type: string
+  default: "Plugwise Thermostat"
+username:
+  description: Should you ever need to change this, you can
+  required: false
+  type: string
+  default: smile
+port:
+  description: When having a custom setup, you can change the port number
+  required: false
+  type: integer
+  default: 80
+min_temp:
+  description: If you want to adjust the lower boundary, the integration will not allow temperatures below the set value.
+  required: false
+  type: integer
+  default: 4
+max_temp:
+  description: If you want to adjust the upper boundary, the integration will not allow temperatures above the set value.
+  required: false
+  type: integer
+  default: 30
+{% endconfiguration %}
+
+### Full configuration example
+
+```yaml
+# Example configuration.yaml entry to show only devices at your vacation and primary homes
+climate:
+  - platform: anna
+    name: Anna Thermostat   # optional, only if you want to use a different name
+    password: your_short_id # required, the ID on the smile (some string of 6 characters)
+    host: local_ip_address  # required, the IP-address of your smile
+    username: smile         # optional, default username is smile
+    port: 80 		    # optional, only needed when other than 80
+    min_temp: 4 	    # optional, when you want to change the minimal temperature
+    max_temp: 30 	    # optional, when you want to change the maximum temperature
+```
+
+### Service
+
+#### Set HVAC mode (schedule)
+
+Service: `climate.set_hvac_mode`
+
+Available options include `auto` or `off`. The meaning of `auto` is that a schedule is active and the thermostat will change presets accordingly. The meaning of `off` there is no schedule active, i.e. the active preset or manually set temperature be used to control the climate of your house/rooms.
```

Comment:
The meaning of `off` is that there is no schedule active, i.e. the active preset or manually set temperature is to be used to control the climate of your house or rooms.